madgiraffestudios/src/components/About/About.scss

```diff
@@ -1,16 +1,18 @@
-#headDiv {
+
+body{
     background-color: grayscale($color: #f8f9fa);
```

Comment:
@MrGDS2 why is this grayscale added to the entire body of the app, it will affect landing page

Reply:
resolved